wavefront/server/plugins/datasource/datasource/bigquery/__init__.py

```
@@ -72,6 +72,17 @@ def insert_rows_json(self, table_name: str, data: List[Dict[str, Any]]):
         result = self.client.insert_rows_json(f'{self.table_prefix}{table_name}', data)
         return result
 
+    async def update_rows_json(
+        self, table_name: str, filter: Dict[str, Any], data: Dict[str, Any]
```

Ruff (0.15.9)

[error] Function argument `filter` is shadowing a Python builtin

(A002)